source/IntegrationTests/Behaviours/IncomingRequests/GivenAggregatedMeasureDataV2RequestWithDelegationTests.cs

```diff
@@ -228,116 +253,132 @@ await ThenNotifyAggregatedMeasureDataDocumentIsCorrect(
+                SenderId: DataHubDetails.DataHubActorNumber,
                 EnergySupplierNumber: energySupplierNumber,
                 BalanceResponsibleNumber: balanceResponsibleParty,
-                SettlementMethod: SettlementMethod.Flex,
-                MeteringPointType: MeteringPointType.Consumption,
-                GridAreaCode: "512",
-                OriginalTransactionIdReference: TransactionId.From("12356478912356478912356478912356478"),
-                ProductCode: "8716867000030", // Example says "8716867000030", but document writes as "5790001330590"?
+                SettlementMethod: testMessageData.ExampleMessageData.SettlementMethod,
+                MeteringPointType: testMessageData.ExampleMessageData.MeteringPointType,
+                GridAreaCode: gridAreaWithDelegation,
+                OriginalTransactionIdReference: transactionId,
+                ProductCode: ProductType.EnergyActive.Code,
                 QuantityMeasurementUnit: MeasurementUnit.KilowattHour,
-                CalculationVersion: GetNow().ToUnixTimeTicks(),
-                Resolution: Resolution.Hourly,
+                CalculationVersion: testMessageData.ExampleMessageData.Version,
+                Resolution: testMessageData.ExampleMessageData.Resolution,
                 Period: new Period(
-                    CreateDateInstant(2024, 1, 1),
-                    CreateDateInstant(2024, 1, 31)),
-                Points: TimeSeriesPointAssertionInput.From(aggregatedMeasureDataRequestAcceptedMessage.Series.Single().TimeSeriesPoints)));
+                    CreateDateInstant(2022, 01, 12),
+                    CreateDateInstant(2022, 01, 13)),
+                Points: testMessageData.ExampleMessageData.Points));
     }
 
     /// <summary>
     /// Rejected document based on example:
     ///     https://energinet.sharepoint.com/:u:/r/sites/DH3ART-team/Delte%20dokumenter/General/CIM/CIM%20XSD%20-%20XML/XML%20filer/XML%2020220706%20-%20Danske%20koder%20-%20v.1.5/Reject%20request%20aggregated%20measure%20data.xml?csf=1&web=1&e=F5bcPI
     /// </summary>
-    [Theory(Skip = "not updated yet")]
-    [MemberData(nameof(DocumentFormatsWithAllRoleCombinations), MemberType = typeof(GivenAggregatedMeasureDataRequestWithDelegationTests))]
-    public async Task AndGiven_DelegationInOneGridArea_AndGiven_InvalidRequest_When_DelegatedActorPeeksAllMessages_Then_ReceivesOneRejectAggregatedMeasureDataDocumentsWithCorrectContent(DocumentFormat incomingDocumentFormat, DocumentFormat peekDocumentFormat, ActorRole delegatedFromRole, ActorRole delegatedToRole)
+    [Theory]
+    [MemberData(
+        nameof(DocumentFormatsWithAllRoleCombinations),
+        MemberType = typeof(GivenAggregatedMeasureDataV2RequestWithDelegationTests))]
+    public async Task
+        AndGiven_DelegationInOneGridArea_AndGiven_InvalidRequest_When_DelegatedActorPeeksAllMessages_Then_ReceivesOneRejectAggregatedMeasureDataDocumentsWithCorrectContent(
```

Comment:
```suggestion
    public async Task
        AndGiven_DelegationInOneGridArea_AndGiven_InvalidRequest_When_DelegatedActorPeeksAllMessages_Then_ReceivesOneRejectAggregatedMeasureDataDocumentsWithCorrectContent(
```
Is this two lines ? :(